pkgs/development/tools/literate-programming/Literate/default.nix

```diff
@@ -1,16 +1,16 @@
-{ lib, stdenv, fetchgit, dmd, dub }:
+{ lib, stdenv, fetchgit, ldc, dub }:
 
 stdenv.mkDerivation {
   pname = "Literate";
-  version = "unstable-2020-09-02";
+  version = "unstable-2021-01-22";
 
   src = fetchgit {
```

Comment:
That should use fetchFromGitHub.

Reply:
This fetches a submodule.

Reply:
`fetchSubmodule = true` also works for `fetchFromGitHub`…